src/components/layout/areas/FeatureTreePane.tsx

```diff
@@ -462,7 +462,7 @@ const OperationItemWrapper = ({
               </code>
             </>
           ) : (
-            <span className="text-sm">{name}</span>
+            <span className="text-sm">{variableName ?? name}</span>
```

Comment:
This is the one-line change: if a variable name is available, we use it, otherwise we default to the operation name